locale/es/docs/index.md

```diff
@@ -7,10 +7,11 @@ labels:
 
 # Sobre la documentación
 
-Existen tres tipos de documentación disponible en este sitio:
+Existen diferentes tipos de documentación disponible en este sitio:
 
 * Referencia de la API
 * Características de ES6
+* Guías
 
 ### Referencia de la API
 
```

Comment:
At line 19 --> 'qué errores...'

Reply:
At 21 (sorry I can't add comment on those lines)
Don't sound so well 'mas no documenta', maybe 'pero no documenta'. What do you think?

Reply:
I agree! That is what is on the current version but makes more sense as you put it, I'll update it